Umbraco-Heartcore/Getting-Started-Cloud/GraphQL-Playground/index.md

```diff
@@ -0,0 +1,95 @@
+---
+versionFrom: 8.0.0
+meta.Title: "Umbraco Heartcore GraphQL Playground"
+meta.Description: "Documentation for Umbraco Heartcore GraphQL Playground"
+---
+
+# GraphQL Playground
+
+With GraphQL query language, you will be able to limit the amount of data transferred to and between your Heartcore instance and the client by specifying exactly what kind of information you would like to get in your API calls.
```

Comment:
With the GraphQL query language

Reply:
data transferred to and from your Heartcore

Reply:
Thanks, added your suggestions in a new commit